etl/steps/data/grapher/countries/2023-09-29/cow_ssm.meta.yml

```diff
@@ -3,5 +3,5 @@ tables:
     variables:
       is_present:
         title: Sovereign state
-        description_short: Whether the country is present in the COW SSM dataset.
+        description_short: The country is identified as a sovereign state by Correlates of War - State System Membership dataset.
```

Comment:
honestly feel free to ignore ofc but again I think it'd be good to note something about the classification criteria of this source in the subtitle - e.g., `Under the Correlates of War criteria, pre-1919 sovereign state status required over 500,000 people and Franco-British recognition; post-1919, it needed League of Nations or UN membership, or similar population size with recognition from two major powers.`

Reply:
this is already captured in `definitions.common.description_key`